README.md

```diff
@@ -39,3 +39,28 @@ You can use composer to bring in modules and themes.  Just run:
 composer require drupal/ctools
 ```
 Contributed modules and themes are .gitignored by default, meaning you need to run `gulp install` each time you clone the repository down.
+
+Using the Docker Images
+-----------------------
+
+#### Default configuration
+
+To use the standard configuration, just run `docker-compose up`. This makes the site available behind Varnish at [http://localhost/8080](http://localhost:8080).
+
+#### Development configuration
+
+The default setup only exposes port 80 from varnish to the outside world (exposed as port 8080 to the host). This is great for production-like environments, but not for development where you may not want a reverse proxy, or you may need to connect directly to MySQL using a tool like Sequel Pro.
+
+The standard `docker-compose.yml` can be overridden/added to using a `docker-compose.override.yml` file (which is ignored by the repository). This project includes an example override file that exposes ports to all relevant containers for direct access from the host in `docker-compose-development.override.yml`. Unless additional customization is needed, it is most likely enough to symlink `docker-compose-development.override.yml` to `docker-compose.override.yml`
```

Comment:
What do you think about just recommending this method:

`docker-compose up -f docker-compose.yml -f docker-compose-development.override.yml`

This accomplishes the same thing as the docker-compose.override.yml without requiring symlinks and whatnot.